### ISSUE TITLE
Pressing `PgUp` in the new App Menu's qubes list's app list breaks keyboard navigation

### ISSUE THREAD

**Issue by UndeadDevel** [QubesOS/qubes-issues#9008]
### Qubes OS release
4.2

### Brief summary
In the new App Menu, using `PgUp` to navigate via the keyboard works in almost all lists (just like `Home`, `End` and `PgDown`), except for the app lists of the qubes list, where it will deselect all items, such that there is no focused item for further navigation, i.e. it breaks keyboard navigation until either the search is used by starting to type something or the mouse is used to restore a selection.

In some other lists `PgUp` will also not select the top item, but for those cases so far navigation is not broken, so `Down Arrow` will select the top item after `PgUp`.

### Steps to reproduce
1. Select and item in a qubes list app list, move the mouse away from the app menu and press `PgUp`.
2. Observe that no further use of the arrow keys or the other navigation keys will select an item.

### Expected behavior
`PgUp` behaves consistently, i.e. in this case selects the top item of the list.

### Actual behavior
Deselects all items, breaking keyboard navigation.

**Comment by marmarta** [comment]
Note for fixing this: this effect only happens if the top application listrow (in the list of all applications) has never been mapped before.

**Comment by marmarta** [comment]
Ok, cant find a way to fix it in a sensible way that works always, but I will push a slight improvement (now it will only break the first time) and note reference to Gtk bug here.
